docs/shared/react-tutorial/react-monorepo.md

```diff
@@ -0,0 +1,827 @@
+import { StrictMode } from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import ReactDOM from 'react-dom/client';
+
+import App from './app/app';
+
+const root = ReactDOM.createRoot(
+  document.getElementById('root') as HTMLElement
+);
+
+root.render(
+  <StrictMode>
+    <BrowserRouter>
+      <App />
+    </BrowserRouter>
+  </StrictMode>
+);
+```
+
+Then we can import the `ProductList` component into our `app.tsx` and render it via the routing mechanism whenever a user hits the `/products` route.
+
+```tsx {% fileName="src/app/app.tsx" %}
+import { Route, Routes } from 'react-router-dom';
+
+// importing the component from the library
+import { ProductList } from '@react-monorepo/products';
+
+function Home() {
+  return <h1>Home</h1>;
+}
+
+export function App() {
+  return (
+    <Routes>
+      <Route path="/" element={<Home />}></Route>
+      <Route path="/products" element={<ProductList />}></Route>
+    </Routes>
+  );
+}
+
+export default App;
+```
+
+Serving your app (`nx serve myreactapp`) and then navigating to `/products` should give you the following result:
+
+![products route](/shared/images/tutorial-react-standalone/react-tutorial-products-route.png)
+
+Let's apply the same for our `orders` library.
+
+- generate a new component `OrderList` in `libs/orders` and export it in the corresponding `index.ts` file
+- import it into the `app.tsx` and render it via the routing mechanism whenever a user hits the `/orders` route
+
+In the end, your `app.tsx` should look similar to this:
+
+```tsx {% fileName="src/app/app.tsx" %}
+import { Route, Routes } from 'react-router-dom';
+import { ProductList } from '@react-monorepo/products';
+import { OrderList } from '@react-monorepo/orders';
+
+function Home() {
+  return <h1>Home</h1>;
+}
+
+export function App() {
+  return (
+    <Routes>
+      <Route path="/" element={<Home />}></Route>
+      <Route path="/products" element={<ProductList />}></Route>
+      <Route path="/orders" element={<OrderList />}></Route>
+    </Routes>
+  );
+}
+
+export default App;
+```
+
+## Building the App for Deployment
+
+<!-- {% video-link link="https://youtu.be/OQ-Zc5tcxJE?t=856" /%} -->
+
+If you're ready and want to ship your application, you can build it using
+
+```{% command="npx nx build myreactapp" path="react-monorepo" %}
+vite v4.3.5 building for production...
+✓ 33 libs transformed.
+dist/myreactapp/index.html                   0.48 kB │ gzip:  0.30 kB
+dist/myreactapp/assets/index-e3b0c442.css    0.00 kB │ gzip:  0.02 kB
+dist/myreactapp/assets/index-378e8124.js   165.64 kB │ gzip: 51.63 kB
+✓ built in 496ms
+
+ ——————————————————————————————————————————————————————————————————————————————————————————————————————————
+
+ >  NX   Successfully ran target build for project reactutorial (1s)
+```
+
+All the required files will be placed in the `dist/myreactapp` folder and can be deployed to your favorite hosting provider.
+
+## Visualizing your Project Structure
```

Comment:
I would show the entire monorepo. THe graph right now only shows the `myreactapp` but doesn't show the `second-app`. I'd have at least a structure where we have a monorepo with these two apps and some libs that are just used by one app and some by both (e.g. the shared-ui for instance could be used by both). 

Or if we have these two `orders` and `products` we could have a second application `vendors` or `admin` that also imports `products` as a hypothetical app which manages the products on this store.